libs/runtime/ebpf_platform.c

```diff
@@ -68,7 +62,11 @@ ebpf_is_preemptible()
 uint32_t
 ebpf_get_current_cpu()
 {
-    return KeGetCurrentProcessorNumberEx(NULL);
+    PROCESSOR_NUMBER processor_number;
+
+    KeGetCurrentProcessorNumberEx(&processor_number);
+
+    return KeGetProcessorIndexFromNumber(&processor_number);
```

Comment:
As discussed offline, the KeGetCurrentProcessorIndex is simpler and higher performance.

Reply:
And not documented yet.